packages/victory-core/src/victory-util/add-events.js

```diff
@@ -192,8 +238,10 @@ export default (WrappedComponent, options) => {
           Events.getPartialEvents(baseEvents, key, componentProps),
           componentProps.events
         );
+        this.globalEvents = Events.getGlobalEvents(events);
```

Comment:
This happens for every data point, so it could get expensive. I think it could be moved to `getCalculatedValues` instead.

Reply:
Same with the `omit` on the next line.

Reply:
How can I do that? It depends on `component` which isn't known unless `renderContainer` is called. Also, I don't think it's more expensive than `Events.getPartialEvents` or `Events.getEvents` which are both called inside the same handler

Reply:
Oops, you're right. These need to be the partial events. My mistake. Still it seems odd that `this.globalEvents` gets reassigned for every child component. Should global events be limited to "parent" events on the container? If they should be supported on the data and label component, too, then we should be assigning onto rather than overwriting `this.globalEvents`.

Reply:
It only makes sense for containers, so I think I should move that assignment to `renderContainer`